server/slack/commands.js

```diff
@@ -0,0 +1,10 @@
+// All commands are case insensitive
+const Commands = {
+  TIL: ["#til", "til", "#todayilearn", "/til", "/todayilearn"],
```

Comment:
`#todayilearned` instead of `#todayilearn`, maybe? (Will update the corresponding command in the Slack app if you make this change.)